packages/neuron-ui/src/widgets/Dialog/dialog.module.scss

```diff
@@ -49,7 +49,6 @@
   padding: 20px 16px;
   position: relative;
   max-height: calc(100vh - 260px);
```

Comment:
How does it look when the message is long and wrapped so the height is over the `max-height`

Reply:
> How does it look when the message is long and wrapped so the height is over the `max-height`

![image](https://github.com/nervosnetwork/neuron/assets/11756869/48b4db70-644e-4849-917b-47b7f4357c63)

Reply:
It looks good for me because we won't have such a long error message